bigtop-packages/src/rpm/flink/SPECS/flink.spec

```diff
@@ -72,19 +81,38 @@ Some of the key features of Apache Flink includes.
     * Fault-tolerance via Lightweight Distributed Snapshots
     * Hadoop-native YARN & HDFS implementation
 
-# Additions for master-worker configuration #
+%package jobmanager
+Summary: Provides the Apache Flink Job Manager service.
```

Comment:
Its better to keep the description exactly the same between Debian and RPM packages